style.css

```diff
@@ -0,0 +1,140 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap");
+
+body {
+  margin: 0%;
+  padding: 0%;
+  background-color: #e5e5e5;
+}
+
+.My_logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 22px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 4%;
+  padding-left: 8%;
+  padding-right: 8%;
+  align-items: center;
+  padding-bottom: 2%;
+  background-color: white;
+}
+
+.menu_icon {
+  width: 30px;
+  height: 3px;
+  background-color: #6070ff;
+  margin: 8px 0;
+}
+
+.Body_content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+  position: absolute;
+  background-image: url(images/background_imag_block_1.png);
+  background-position: 100%;
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-bottom-left-radius: 70px;
+  background-color: white;
+  height: 916px;
+}
```

Comment:
- [ ] Sincerely, I really commend your good disposition to learning! As suggested by the last reviewer, you've done well but can we just work more on the background image so that we can have something very close to the figma design. This will really helps your Software Development journey because hiring managers are always on the look out to those that can replicate a given template. 👍 I tried to re-adjust your piece of code below, you may like to use it. The `background-position`, `background-size` and `height` of the container is interfering with the image.

```suggestion
.Body_content {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  padding: 0;
  gap: 12px;
  position: absolute;
  background-image: url(images/background_imag_block_1.png);
  background-repeat: no-repeat;
  border-bottom-left-radius: 70px;
  background-color: white;
}
```

## Your Design
![Untitled](https://user-images.githubusercontent.com/103386183/220117363-ed712162-e68a-483f-95d0-5880a96673c3.jpg)

## Figma Design
![Untitled](https://user-images.githubusercontent.com/103386183/220119467-6f3895a5-e55e-4d1c-b006-5aaf6726dcf0.jpg)

Reply:
Sir can we have 5-minute zoom meeting. Please Adjust your time and let we have a zoom meeting.

Reply:
![image](https://user-images.githubusercontent.com/102411286/220135060-6fb49794-fa69-457f-92fe-45a1ba68dcf0.png)

This Output is my CSS code, 

have You seen the difference... Please sir, I need your time and let we have a 5 min zoom meeting.

Reply:
@WasyihunS, so sorry for late response I just saw your message. Next time you supposed to have tagged me for a quick response. Yes, the image you added to the comment section shows that background-image is okay like what we have in the design. The only **optional** issue is that you can reduce the spacing between the elements. It's **optional** because the previous reviewer did not mention it.

Reply:
@OyePriscilla 
Thank you for the response, as you see, I have used all my effort to design this project as a beginner. 
and I don't understand where my mistake is???

That is why I am asking you to have 5 min zoom meeting.

Reply:
@WasyihunS, that's fine. Can we make it 12:00pm program time during lunch break?

Reply:
@OyePriscilla Hello Dear, I am waiting you on Zoom now

Reply:
@OyePriscilla Hello Dear, Is there a problem, The meeting Was canceled.

Reply:
@WasyihunS, it's from your end I think.